Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1403,4 +1403,49 @@ public function isMonolithicPhp()
 
         return 0 === count($kinds[T_INLINE_HTML]) + count($hhvmHashBangs) && 1 === count($kinds[T_OPEN_TAG]) + count($kinds[T_OPEN_TAG_WITH_ECHO]) + count($hhvmOpenTagsWithEcho);
     }
+
+    /**
+     * Clear tokens at given indexes.
+     *
+     * Merges whitespace tokens around the cleared tokens if needed
+     *
+     * @param int[] $indexes
+     */
+    public function clearTokens(array $indexes)
```

Comment:
>clearRange
from - till

>clearTokens
individual indexes

or do you mean changing `clearRange` to merge possible whitespace tokens as well?